lmfit/jsonutils.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+"""
+ json utilities for larch objects
+"""
+import sys
+import six
+import json
+import numpy as np
+from base64 import b64encode, b64decode
+
+try:
+    import dill
+    HAS_DILL = True
+except ImportError:
+    HAS_DILL = False
+
+try:
+    from pandas import DataFrame, Series, read_json
+except ImportError:
+    DataFrame = Series, type(NotImplemented)
```

Comment:
you cannot use ```Series``` here - you check whether ```pandas``` is available, but even if it's not you still try to use ```Series```. It passes the tests, because there we do install ```pandas```. I will update this in #443 probably so that for the ```minimum``` we only install packages listed in ```requirements.txt```; alternatively, if you need to use this we should require ```pandas```.